code/src/Components/ThoughtElements.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Moment from 'moment';
```

Comment:
Consider using another library, like date-fns, since Moment was deprecated and will eventually become obsolete.